pyproject.toml

```diff
@@ -1,18 +1,25 @@
 [build-system]
-requires = ["setuptools>=61", "cffi>=1.3.0", "requests", "setuptools-scm", 'asn1crypto']
-build-backend = "setuptools.build_meta"
+requires = [
+    'scikit-build-core>=0.9.0',
+    'hatchling',
+    'setuptools>=42.0.0',
+    'cffi>=1.3.0,
```

Comment:
```suggestion
    'cffi>=1.3.0',
```

Reply:
Right, Thanks, I got this one and corrected it at my end, but realized the `tox` test did not pass at my end either (I was using `python -m build`), so I stopped committing failing updates to minimize the emails